### ISSUE TITLE
Contrast for DEV User Liquid tag lacking contrast in night theme

### ISSUE THREAD

**Issue by mariocsee** [thepracticaldev/dev.to#2847]
**Describe the bug**
The DEV User Liquid tag is really hard to read and discern when in night theme.

**To Reproduce**

1. Switch to night theme on DEV
2. Click on [this post, for example](https://dev.to/jess/who-in-your-life-deserves-a-shout-out-right-now-4p2o)
3. See tagged users below and that the username + description is hard to read because of the font color

**Expected behavior**
Font color or background color should be adjusted for readability.

**Screenshots**
<img width="815" alt="Screen Shot 2019-05-15 at 17 02 26" src="https://user-images.githubusercontent.com/13403332/57809513-98aca700-7733-11e9-898f-5ce16ec6ae38.png">

It's quite hard to read about rhymes here.

**Desktop (please complete the following information):**

- OS: MacOS Mojave
- Browser Chrome
- Version 73

**Additional context**
[Ben's post about themes on DEV](https://dev.to/devteam/contributors-wanted-night-mode-themes-on-dev-i8n)

**Comment by ashleemboyer** [comment]
I'm having a similar contrast issue when inline code is displayed:
![image](https://user-images.githubusercontent.com/43934258/57823481-b3980f00-7765-11e9-9586-68e7503c190c.png)

**Comment by maestromac** [comment]
Thanks for pointing that out @ashleemboyer . I believe that's a separate issue.
